tests/Temporalio.Tests/Extensions/Hosting/TemporalWorkerServiceTests.cs

```diff
@@ -1,3 +1,5 @@
+using Temporalio.Tests.Worker;
```

Comment:
Don't usually have `using` above the namespace here and would prefer to just copy that simple workflow into here instead of have test files depend on each other.